### PR TITLE
CORTX-29573:rgw_setup config failed with object has no attribute decode

### PR DESCRIPTION
**Pull request by shriya-deshmukh**
Signed-off-by: Shriya Deshmukh <shriya.deshmukh@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: src/rgw/const.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 SERVICE_NAME = f'{COMPONENT_NAME}_setup' # rgw_setup
 INSTALL_PATH = '/opt/seagate/cortx'
 RGW_INSTALL_PATH = f'{INSTALL_PATH}/{COMPONENT_NAME}'
-ADMIN_CREATION_TIMEOUT = 60
+ADMIN_CREATION_TIMEOUT = 600
```
Comment (cdeshmukh) [inline src/rgw/const.py:24]: Add 
#TODO Revisit after Motr delay issue resolved, seen while admin creation, for CORTX cluster with 15 nodes

Comment (shriya-deshmukh) [reply src/rgw/const.py:25]: Done.

**Review by sachitanands — APPROVED** [review]
looks good

**Review by suryakumar1024 — APPROVED** [review]
LGTM

**Pull request merged**